src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/CodeStyle/CSharpCodeStyleSettingsProvider.cs

```diff
@@ -15,6 +15,8 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.CSharp.EditorConfigSettings.DataProvider.CodeStyle
 {
+    using Workspace = Microsoft.CodeAnalysis.Workspace;
```

Comment:
why this change?

Reply:
To call LSP apis we need to bring in `Microsoft.VisualStudio.LanguageServer.Client.Implementation` that unfortunately brings in types defined in the `Microsoft.VisualStudio.Workspace`. This causes almost all uses of the `Workspace` type to be made ambiguous. Talked to @gundermanc about getting the types we need moved to a reference assembly so we don't have to bring in all these types. We want to ensure that we don't merge this into main, but with unblocks work for now.